packages/builder/src/components/design/settings/controls/FieldConfiguration/FieldConfiguration.svelte

```diff
@@ -43,6 +49,10 @@
     cachedValue = cloneDeep(value)
   }
 
+  $: if (typeof $updates.resp == "string") {
```

Comment:
NAB: How would this differ from:

```js
if ($updates.resp) {
  updating = false
}
```

Reply:
They achieve the same thing, but the `typeof` just communicates the contains of `resp` better